docs/application/native/guides/device/battery-monitor.md

```diff
@@ -0,0 +1,230 @@
+
+1. Fetching information for an application for a particular resource over certain duration-
+   The `battery_monitor_resource_id_e`, `battery_monitor_duration_type_e` enums can be used to provide information related to specific resource and particular duration type respectively, for example:
+
+   ```
+   #include <battery_monitor.h>
+
+   void
+   func(void)
+   {
+	char* app_id = "org.tizen.samsung";
+
+	int error_code = BATTERY_MONITOR_ERROR_NONE;
+	int battery_usage = -1;
+	battery_monitor_resource_id_e resource_id = BATTERY_MONITOR_RESOURCE_ID_DISPLAY;
+	battery_monitor_duration_type_e duration_val = BATTERY_MONITOR_DURATION_TYPE_1DAY;
+
+	error_code = battery_monitor_get_usage_by_app_id_for_resource_id(app_id, resource_id, duration_val, &battery_usage);
+
+	if (error_code == BATTERY_MONITOR_ERROR_NONE)
+		printf("The Battery Usage for appid [%s], for resource display is [%d]", app_id, battery_usage);
+	else
+		printf("Error Occurred [%d]", error_code);
+
+	resource_id = BATTERY_MONITOR_RESOURCE_ID_WIFI;
+	duration_val = BATTERY_MONITOR_DURATION_TYPE_1WEEK;
+
+	error_code = battery_monitor_get_usage_by_app_id_for_resource_id(app_id, resource_id, duration_val, &battery_usage);
+
+	if (error_code == BATTERY_MONITOR_ERROR_NONE)
+		printf("The Battery Usage for appid [%s], for resource wifi is [%d]", app_id, battery_usage);
+
+	else
+		printf("Error Occurred [%d]", error_code);
+
+	free(app_id);
+
+	return;
+   }
+   ```
+
+2. Fetching total Battery Usage information of Application Id, combining all the resources over certain duration-
+   The `battery_monitor_duration_type_e` enum can be used to choose a particular duration type, for example:
+
+   ```
+   #include <battery_monitor.h>
+
+   void
+   func(void)
+   {
+	char* app_id = "org.tizen.samsung";
+
+	int error_code = BATTERY_MONITOR_ERROR_NONE;
+	int battery_usage = -1;
+	battery_monitor_duration_type_e duration_val = BATTERY_MONITOR_DURATION_TYPE_1DAY;
+
+	error_code = battery_monitor_get_total_usage_by_app_id(app_id, duration_val, &battery_usage);
+	if (error_code == BATTERY_MONITOR_ERROR_NONE)
+		printf("The total battery usage for appid [%s] over last 24 hrs is [%d]", app_id, battery_usage);
+	else
+		printf("Error Occurred [%d]", error_code);
+
+	duration_val = BATTERY_MONITOR_DURATION_TYPE_1WEEK;
+	error_code = battery_monitor_get_total_usage_by_app_id(app_id, duration_val, &battery_usage);
+	if (error_code == BATTERY_MONITOR_ERROR_NONE)
+		printf("The total battery usage for appid [%s] over last 7 days is [%d]", app_id, battery_usage);
+	else
+		printf("Error Occurred [%d]", error_code);
+
+	return;
+   }
+   ```
+3. Fetching Battery Usage values for all the resources used by an Application Id for a certain duration-
+   The `battery_monitor_resource_id_e`, `battery_monitor_duration_type_e` enums can be used to provide information related to specific resource and particular duration type respectively, for example:
```

Comment:
The `battery_monitor_resource_id_e` and the  `battery_monitor_duration_type_e` enums can be used to provide information related to a specific resource and a particular duration type respectively, as shown in the following code: